### PR TITLE
11 get reviews queries

### PR DESCRIPTION
**Pull request by JJW-27**

### CODE REVIEW

File: models/reviews.models.js [hunk 1]
```diff
@@ -1,30 +1,54 @@
...
+    'title',
+    'designer',
+    'owner',
+    'category',
+    'created_at',
+    'votes',
+  ];
+
+  const validSorts = ['DESC', 'ASC'];
+
```
Comment (yangalexyangg) [inline models/reviews.models.js:15]: Do you have anything that validates the value for ORDER BY to prevent injection? I can you do for SORT BY

**Review by yangalexyangg — APPROVED** [review]
Great progress Joe, just need to look at my comment but I will approve anyway

File: models/reviews.models.js (hunk 1 above)
Comment (JJW-27) [reply models/reviews.models.js:15]: Yeah it was just above it

Comment (JJW-27) [reply models/reviews.models.js:15]: Also - thank you!

**Pull request merged**